.rubocop.yml

```diff
@@ -1,3 +1,4 @@
-Lint/AmbiguousBlockAssociation:
-  Exclude:
-    - "spec/**/*"
```

Comment:
D'oh, I didn't spot that I'd changed this.  It's the reason I had to make fixes in spec/unit/lib/gdpr/gateway/user_details_spec.rb.  Probably better that we avoid unnecessary exclusions though; I can't see an obvious reason not to just use the style that the linter prefers in this case.